src/types/beneficiary.ts

```diff
@@ -0,0 +1,56 @@
+export interface Props {
```

Comment:
Everything in here was already added but it now looks like you did add them as new items 

This PR needs cleaning

Sync with upstream and make sure this Par ONLY has things relating to the table you are putting up and nothing else

Reply:
- [x] removed item